mpas_analysis/test/test_climatology.py

```diff
@@ -321,7 +321,13 @@ def test_cache_climatologies(self):
 
         remapper = self.setup_mpas_remapper(config)
 
-        tests = []
+        return {'config': config, 'calendar': calendar, 'ds': ds,
+                'fieldName': fieldName, 'climFileName': climFileName,
+                'refClimatology': refClimatology, 'remapper': remapper}
+
+
+    def test_jan_1yr_climo_test1(self):
```

Comment:
Just split up each test into its own test and make sure each was initalized on its own to help avoid accidental side effects.